index.html

```diff
@@ -747,10 +757,29 @@ <h2>
                   </li>
                 </ol>
               </li>
-              <li>Wait for a significant change of geographic position. What
-              constitutes a significant change of geographic position is left
-              to the implementation. User agents MAY impose a rate limit on the
-              frequency position changes.
+              <li>
+                <span id="wait-for-change">Wait for a significant change of
+                geographic position</span>. What constitutes a significant
+                change of geographic position is left to the implementation.
+                User agents MAY impose a rate limit on the frequency position
+                changes.
+              </li>
+              <li>If |document| is not [=Document/fully active=] or not
+              [=Document/visible=], go back to the previous step and again
```

Comment:
Do we want to remember that it happened (not the exact position, but just that we dropped an update) so that we can send an update with the current position when the document becomes fully active and visible again? Otherwise when a page is restored from bfcache and the user stopped moving (?) it might be stuck with a stale location.

Reply:
In w3ctag/design-principles#317 the example explicitly calls out the Geolocation API as a case where updates should not be queued and delivered once the document becomes fully active. It is however important to note that a site which has called `watchPosition()` expects geolocation updates when there is a significant change in position. To do otherwise will likely cause the site to appear stale when it is restored from bfcache and result in a poor user experience. Therefore, while updates SHOULD NOT be queued so that the site can observe the history of location changes while it was not fully active a single update SHOULD be delivered if the location is significantly different from the last value the site received while it was fully active.

Reply:
Yep that's what I wanted to say. I guess we need to track the last position when the page becomes not fully active / visible then, so that we can compare it to the position when it becomes fully active & visible again?

Reply:
>  watchPosition() expects geolocation updates when there is a significant change in position.

That's true, but *iff* the document is visible (and fully active). 

If the document transitions from "hidden" to "visible", it's perfectly reasonable for the page to call `getCurrentPosition()` to get a position update. But, `.watchPosition()` must only receive *new significant position changes* when visible. That is, if the page was hidden and there was a significant update, then that's too bad/too late - it has to wait for the next significant change.

See code below.

> Yep that's what I wanted to say. I guess we need to track the last position when the page becomes not fully active / visible then, so that we can compare it to the position when it becomes fully active & visible again?

My opinion is that we don't need to go to this level, because it can be handled by the developer:

```JS
// handle switching tabs
window.onvisibilitychange = () => {
   if (document.visibilityState === 'visible') {
    navigator.geolocation.getCurrentPosition(updateMap);
  }
}
// keep getting significant updates indefinitely...
navigator.geolocation.watchPosition(updateMap);
```

Reply:
> My opinion is that we don't need to go to this level, because it can be handled by the developer:

I guess that sounds reasonable, for the bfcache case web devs can listen for the `pageshow` event as well. I guess I'm a bit worried that most web devs are not aware of bfcache and thus might miss this edge case, but I don't really have a strong opinion on this so I'll trust your decision here!